index.bs

```diff
@@ -3918,12 +3918,14 @@ the attestation=] is consistent with the fields of the attestation certificate's
         public key in the first certificate in |x5c| with the algorithm specified in |alg|.
     - Verify that the public key in the first certificate in |x5c| matches the
         <code>[=credentialPublicKey=]</code> in the <code>[=attestedCredentialData=]</code> in |authenticatorData|.
-    - Verify that in the attestation certificate extension data:
-        - The value of the `attestationChallenge` field is identical to |clientDataHash|.
-        - The `AuthorizationList.allApplications` field is <em>not</em> present, since PublicKeyCredential MUST be [=scoped=] to the
-            [=RP ID=].
-        - The value in the `AuthorizationList.origin` field is equal to `KM_TAG_GENERATED`.
-        - The value in the `AuthorizationList.purpose` field is equal to `KM_PURPOSE_SIGN`.
+    - Verify that the `attestationChallenge` field in the attestation certificate extension data is identical to |clientDataHash|.
+    - Verify the following using the appropriate authorization list from the attestation certificate extension data:
```

Comment:
where is the "attestation certificate extension data" defined? I could not easily find it...   @sbweeden noted this in issue #1022:

It would be useful to point out that this extension data is oid: "1.3.6.1.4.1.11129.2.1.17"

If the above is correct, then I suggest we add it using a sentence ( e.g. in a separate subsection a la https://w3c.github.io/webauthn/#tpm-cert-requirements ?) sorta like so:

```
### Android Key Attestation Statement Certificate Requirements ### {#key-attstn-cert-requirements}

Android Key Attestation [=attestation certificate=]'s 
<dfn>android key attestation certificate extension 
data</dfn> is identified by the OID  
"1.3.6.1.4.1.11129.2.1.17".
```

And then where "extension data" term is used below, substitute `[=android key attestation certificate extension data|extension data=]`

Reply:
Done, thank you Jeff!

The Attestation extension schema is fairly clearly documented at https://source.android.com/security/keystore/attestation, which I have set as the defining link at the beginning of this section. I'm trying to avoid referring to specific sections or language there, since it is a living document.

Reply:
Leaving this open, please resolve if you're ok with the changes.